### PR TITLE
Changed title color to darkblue.

### PR DESCRIPTION
**Pull request by anaferreira7**
Fixed issue related to #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <body>
     <header>
       <img id="logo" src="logo.png" alt="A logo that contains the word 'Git'" />
-      <h1>Git & GitHub Crash Course</h1>
+      <h1 style="color: darkblue">Git & GitHub Crash Course</h1>
```
Comment (PedroAlmeidacode) [inline index.html:13]: You could use a class instead

**Review by PedroAlmeidacode — CHANGES_REQUESTED** [review]
Please solve the comments

File: index.html (hunk 1 above)
Comment (anaferreira7) [reply index.html]: > You could use a class instead

Sure. I'll update it asap.

**Review by anaferreira7 — COMMENTED** [review]
Commited the improvements related to #2

File: index.html (hunk 1 above)
Comment (PedroAlmeidacode) [reply index.html]: That is nice, I will approve it

**Review by PedroAlmeidacode — APPROVED** [review]
Very nice

**Comment by PedroAlmeidacode** [comment]
Closing issue #1

**Pull request merged**